django_tables2/columns/templatecolumn.py

```diff
@@ -14,7 +15,9 @@ class TemplateColumn(Column):
     Arguments:
         template_code (str): template code to render
         template_name (str): name of the template to render
-        extra_context (dict): optional extra template context
+        context_object_name (str): name of the context variable to pas the record in, defaults to "record".
```

Comment:
```suggestion
        context_object_name (str): name of the context variable to pass the record in, defaults to "record".
```

Reply:
"to pass the record in" klinkt gek. Wellicht "name of the context variable that represents the record, defaults to "record". ?

Reply:
```suggestion
        context_object_name (str): name of the context variable that represents the record, defaults to "record".
```